### ISSUE TITLE
[Package Request]: rpcs3

### ISSUE THREAD

**Issue by soredake** [microsoft/winget-pkgs#35207]
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: rpcs3 team
Publisher Url: https://rpcs3.net
Package Name: rpcs3
Description: The Open-source PlayStation 3 Emulator
Package Version: 0.0.19
Installer URL: https://github.com/RPCS3/rpcs3-binaries-win/releases/download/build-de2be9063655b5a92a9acbb19019320c62a434ca/rpcs3-v0.0.19-13027-de2be906_win64.7z

Current Package Identifier (for new versions):

**Comment by Trenly** [comment]
![115ADE10-8019-4638-9BAF-D5B8A58FC75A](https://user-images.githubusercontent.com/12611259/142399685-c980c327-bfb3-4097-830c-0a05dd40fc4c.jpeg)

.7z is a form of zipped or compressed file. This is blocked by #174

**Comment by Trenly** [comment]
Even though .zip is unblocked, .7z was not part of the initial implementation. This is still blocked by #2899

**Comment by Trenly** [comment]
[Policy] Portable Archive

**Comment by DandelionSprout** [comment]
Seems to still rely on .7z almost 4 years later according to https://github.com/RPCS3/rpcs3-binaries-win/releases/ .

The more I think about it, the more I honestly don't understand how the "Support installing other archive formats` PR has been taking several years so far.

**Comment by Dragon1573** [comment]
As I know, `explorer.exe` only supports `.zip` archive. CLI team might not want to introduce third-party archive extractor.

**Comment by DandelionSprout** [comment]
A Windows 11 update sometime earlier this year (I think it was in March or something?) added supported for at least 7Z and RAR in File Explorer, and I think several other formats as well.

**Comment by tajnymag** [comment]
> > A Windows 11 update sometime earlier this year (I think it was in March or something?) added supported for at least 7Z and RAR in File Explorer, and I think several other formats as well.
> 
> But that would not be supported if anyone was running WinGet on Windows 10, Or Server

Microsoft is killing official support for Windows 10 next month. Is winget going to be supported on an end-of-life OS?
